spec/unit/resources/scope/create_if_missing_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './../../../spec_helper'
+
+describe 'oneview_test_api300_synergy::scope_create_if_missing' do
+  let(:klass) { OneviewSDK::API300::C7000::Scope }
```

Comment:
@tmiotto @jsmartt :warning: I could not get around the error of the create methods calling API 200 and resulting in constant missing. When I explicitly declared the class, it passed though.
Not sure on this, thoughts? Should I let the class explicitly declared as it is or would you guys know how to avoid calling the api 200?